cluster/terraform_kubernetes/grafana.tf

```diff
@@ -207,3 +311,99 @@ resource "kubernetes_ingress_v1" "grafana_ingress" {
     }
   }
 }
+
+resource "random_uuid" "admin_role_id" {}
+resource "random_uuid" "editor_role_id" {}
+resource "random_uuid" "viewer_role_id" {}
+
+resource "azuread_application" "grafana" {
+  display_name     = "Grafana SSO ${var.environment}"
```

Comment:
Can we have the name with s189x01 at the beginning, e.g. s189d01-tsc-grafana-appreg